### ISSUE TITLE
Pre-built .so and .a files

### ISSUE THREAD

**Issue by emileb** [atsushieno/android-fluidsynth#9]
Hi, great work on this!

Would it be possible for you provide the pre-built shared and static libraries for this? 99% of the time this is all we need :)
I did this for Boost:
https://github.com/emileb/Boost-for-Android-Prebuilt

Thank you!

**Comment by atsushieno** [comment]
That's true, though there is no prebuilt static library (not part of the build outcome). The prebuilt shared libraries for armeabi-v7a and x86 are available here: https://www.dropbox.com/s/ja791m9hjkwcnrw/android-fluidsynth-binary.zip?dl=0

After my local OS upgrade I don't have a working build environment right now. Nothing further will be provided anytime soon.

**Comment by emileb** [comment]
Great, thank you for the download link 👍

**Comment by atsushieno** [comment]
The repository is now superceded by https://github.com/atsushieno/fluidsynth-fork/tree/opensles-v2.

I have uploaded a new prebuilt binary based on the new repo: https://www.dropbox.com/s/081mnfzhgavjb0y/android-fluidsynth-opensles-binaries-9a4c265.zip?dl=0

(There is no "issues" on the new repo and I keep it updated here for a while.)
